src/pb_stub.cc

```diff
@@ -1149,6 +1178,18 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
           "send", &ResponseSender::Send, py::arg("response") = nullptr,
           py::arg("flags") = 0);
 
+  py::class_<ResponseGenerator, std::shared_ptr<ResponseGenerator>>(
+      module, "ResponseGenerator")
+      .def(py::init<const std::vector<std::shared_ptr<InferResponse>>&>())
+      .def(
+          "__iter__",
+          [](ResponseGenerator& self) {
+            return py::make_iterator(self.Begin(), self.End());
+          },
+          py::keep_alive<0, 1>())
+      .def("__next__", &ResponseGenerator::Next)
+      .def("__len__", &ResponseGenerator::Length);
```

Comment:
Let's remove `__len__` since it cannot be supported when we change the response delivery in future releases.

Reply:
Removed.